src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
+
+type Colors = 'red' | 'green' | 'blue';
+
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
 export interface Figure {
+  shape: Shape;
+  color: Colors;
+  getArea(): number;
+}
+
+function validateTriangleLength(...shapeSide: number[]): boolean {
```

Comment:
```suggestion
function isNotValidTriangle(...shapeSide: number[]): boolean {
```
As you're returning a boolean value it's better to set proper name.